uninstall.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+set -e
+
+function cleanup() {
+  set +e
+}
+
+function set_variable_from_command() {
+  if [[ ($# -ne 3) ]];
+  then
+    echo "Internal Error - Wrong number of parameters"
+    exit 1
+  fi
+  VAR_NAME=$1
+  DEFAULT_COMMAND=$2
+  if [ -z "${!VAR_NAME}" ]
+  then
+      result=$(bash -c "${!DEFAULT_COMMAND}") || {
+        ret=$?; echo "$3 - Code: $ret"; return $ret;
+      }
+      declare -gx "$VAR_NAME"=$result
+  fi
+  echo "[PRE-UNINSTALL]: $VAR_NAME is set to: ${!VAR_NAME}"
+}
+
+function print_help() {
+   echo "Arguments:"
+   echo "   --help: print this help"
+   echo "   --test: source file only for testing"
+}
+
+function unjoin() {
+  kubectl get clusterconfig configuration -o yaml | sed "s/enableBroadcaster: true/enableBroadcaster: false/g" | kubectl replace -f -
```

Comment:
Two comments here:
1. I would improve the error handling (i.e. is the `clusterconfig` guaranteed to exist named that way in any possible situation?)
2. I would replace this with `kubectl patch` if possible